.sync.yml

```diff
@@ -1,4 +1,12 @@
 ---
 .travis.yml:
+  docker_sets:
+    - set: centos7-64
+    - set: centos6-64
+    - set: debian8-64
+    - set: debian9-64
+    - set: ubuntu1404-64
```

Comment:
Ubuntu 14.04 is EoL, we should not add it.

Reply:
This OS is listed in metadata.json.
My goal was not to modify the list of supported OSes.

Reply:
done